2364/2/parser.rb

```diff
@@ -0,0 +1,64 @@
+require 'russian_obscenity'
+require 'terminal-table'
+load 'participant.rb'
+load 'modules.rb'
+
+# Class parse for parsing data
+class Parser
+  PATH = File.expand_path('.').freeze
+  attr_reader :participants, :methods
+  def initialize(value, name)
+    @participants = []
+    @methods = create_methods
+    create_participants(Dir[PATH + '/rap-battles/*'])
+    @participants.sort_by! { |participant| participant.participant_data[:bad_words] }
+    @participants.reverse!
+    if name
```

Comment:
instead of using separate methods - it should be using separate classes

Reply:
I do not fully understand what this means